contracts/base/AaveV2Wrapper.sol

```diff
@@ -0,0 +1,201 @@
+ * 3 - A 25% loan is borrowed in USDC and transferred to the cruize wallet.
+ */
+contract AaveV2Wrapper is OwnableUpgradeable, ReentrancyGuardUpgradeable {
+    using SafeMath for uint256;
+
+    //----------------------------//
+    //        State Variable      //
+    //----------------------------//
+
+    uint256 public priceFloor;
+    uint256 public constant BASE = 10000;
+
+    address public crContract;
+    address public cruize_wallet;
+    uint256 public borrowRatio = 2500; // 25% of 1000
+    uint256 public constant aaveDefaulLTV = 8250; // 82.5% of 1000
+    uint256 private constant USD_DECIMALS = 8;
+    uint256 private constant ETH_DECIMALS = 18;
+
+    //----------------------------//
+    //        Mappings            //
+    //----------------------------//
+
+    // reserveToken => crToken
+    mapping(address => address) public lpTokens;
+    // reserveToken => oracles
+    mapping(address => address) public oracles;
+
+    //----------------------------//
+    //        Address             //
+    //----------------------------//
+
+    IPoolV2 internal constant POOL =
+        IPoolV2(0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9);
+
+    IWETHGateway internal constant WETHGATEWAY =
+        IWETHGateway(0xcc9a0B7c43DC2a5F023Bb9b738E45B0Ef6B06E04);
+
+    IERC20 internal constant BORROW_ASSET =
+        IERC20(0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48); // USDC
+
+    address public constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address public constant ETH = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+    //----------------------------//
+    //          Events            //
+    //----------------------------//
+
+    event AddAsset(address indexed _asset, address _oracle);
+    event BorrowRatioChanged(uint256 indexed ratio);
+    event WithdrawEvent(
+        address indexed asset,
+        address indexed account,
+        uint256 amount
+    );
+    event DepositEvent(address indexed account, uint256 amount);
+    event CreateToken(
+        address indexed tokenAddress,
+        string tokenName,
+        string tokenSymbol,
+        uint8 decimal
+    );
+
+    //----------------------------//
+    //       View Functions       //
+    //----------------------------//
+
+    /**
+     * @dev fetch the latest price of the asset from oracle
+     * @param oracle address for which we need price
+     */
+    function getLatestPrice(address oracle)
+        internal
+        view
+        returns (int256 price)
+    {
+        AggregatorV3Interface priceFeed = AggregatorV3Interface(oracle);
+        (, price, , , ) = priceFeed.latestRoundData();
+    }
+
+    function normalize(uint256 _amount) internal pure returns (uint256) {
+        return _amount * 10**(ETH_DECIMALS - USD_DECIMALS);
+    }
+
+    function calculateBorrow(address reserve, uint256 amount)
+        internal
+        view
+        returns (uint256)
+    {
+        if (reserve == ETH) reserve = WETH;
```

Comment:
Noticing we're using WETH and ETH directly to make comparison. We need to take WBTC into consideration as well.

Reply:
Actually ETH and WETH have same price so that's why we have to handle them separately, other than these assets like wbtc we can handle them independently.